Automattic/wp-calypso
# A4A: Implement Team member invite form.

This pull request  is for adding the team member invite form.

<img width="1726" alt="Screenshot 2024-08-20 at 4 06 05 PM" src="https://github.com/user-attachments/assets/3d18cf30-fe10-4ab6-8093-d39b4d4d5df1">


Closes https://github.com/Automattic/automattic-for-agencies-dev/issues/948

## Proposed Changes

* Implement the Team member invite form using existing reusable form components.
* Added initial submit mutation hook. (This will be updated once we have the endpoint ready)

## Why are these changes being made?

*

## Testing Instructions

* Use the A4A live link and go to the `/team/invite` page.
* Confirm that the form matches the design and is working as expected. Note that clicking the 'Send invite' button will throw an error as the backend endpoint is not yet ready.

## Pre-merge Checklist
